vllm/engine/local_worker_utils.py

```diff
@@ -0,0 +1,242 @@
+import asyncio
```

Comment:
QQ: this file is probably very specific to mp based executor. Should we create executor/util and move it to there?

Reply:
@rkooo567 yes maybe the naming is bad but that's what it's meant for, equivalent to `ray_utils.py` which is in the same place.